src/models/user.js

```diff
@@ -1,15 +1,54 @@
 import crypto from 'crypto'
 import { Model } from 'sequelize'
+import jwtBuilder from '../JWTBuilder'
+import InvalidCredentialsError from '../errors/InvalidCredentialsError'
 
-export default (sequelize, DataTypes) => {
+export function buildUserModel (sequelizeModels) {
```

Comment:
It's OK if you don't provide a name for this function. And it's OK too if you want to.

Reply:
This function is used in [line 70](https://github.com/nahudalla/chat-wedevelop-training/pull/8/files/78638318df5ea601addb5da90f8bfd9387e7e755#diff-9bb9b97462d75fd266eef5db55a0d6beR70) of this file, that's why I added a name to it.

Reply:
Hmm I see. You wanted to split model's behavior from model's definition. Yet, I don't think this is very clear. Let's discuss it with more people. Reach out Sergio or Mauricio. They may have some ideas for this code (opinions are always very useful).

Reply:
Hmm I see. You wanted to split model's behavior from model's definition. Yet, I don't think this is very clear. Let's discuss it with more people. Reach out Sergio or Mauricio. They may have some ideas for this code (opinions are always very useful).

Reply:
I split it when I started writing [tests for this model](https://github.com/nahudalla/chat-wedevelop-training/pull/8/files#diff-2225ce59b6ca901def280cff2ce3d655R22). At first, I thought I'd need to instantiate this class, without the call to [`User.init(...)`](https://github.com/nahudalla/chat-wedevelop-training/pull/8/files#diff-9bb9b97462d75fd266eef5db55a0d6beR72), to be able to test it. I will remove it since I ended up not using it in the tests for the model.